### ISSUE TITLE
SDK 2.2.403 is missing

### ISSUE THREAD

**Issue by DerAlbertCom** [dotnet/sdk#3950]
Your SDK Upgrade Rules and Policies are getting not better.

There is no SDK 2.2.403 (for VS 16.2 and V16.3) and which includes Runtime 2.2.8. With that in mind that we should not specificy an ASP.NET Core Version in the .csproj means that even with Runtime 2.2.8 installed only Runtime 2.2.7 will be used at compile and debug time. Because without a Version Number on the ASP.NET Core PackageReference the Runtime from the SDK will be used.

So how I am supposed to test against 2.2.8?

**Comment by NetTecture** [comment]
Yeah. I just spend a meeting with all developers in my team trying to sort out whether this is supposed to make sense and under what combination of drugs and booze it actually should do so. We could not come up with anything except someone obviously overlooking to publish the latest packages - for nearly a week now.

It seems 2.2.8 updates are still not published. How are we supposed to work with that? I know 2.2 is nearing end of life, but nearing is not being there and thanks to the odata team not publishing their code for anything after that (yet) - we are stuck on 2.2.
